doc/Makefile

```diff
@@ -217,7 +217,7 @@ latex-build: generate
 	mkdir -p build/latex build/doctrees
 	$(SPHINXBUILD) -b latex $(ALLSPHINXOPTS) build/latex $(FILES)
 	$(PYTHON) postprocess.py tex build/latex/*.tex
-	perl -pi -e 's/\t(latex.*|pdflatex) (.*)/\t-$$1 -interaction batchmode $$2/' build/latex/Makefile
+	perl -pi -e 's/LATEXOPTS =/LATEXOPTS ?= --halt-on-error/' build/latex/Makefile
```

Comment:
It seems the original substitution expression was for an earlier version of this Makefile, since I could not find `halt-on-error` in the `build/latex/Makefile` after running `make latex`. 

+1 for this change